qutebrowser/commands/userscripts.py

```diff
@@ -323,12 +346,41 @@ def __str__(self):
         return "Userscripts are not supported on this platform!"
 
 
+def _lookup_path(cmd):
+    """Search userscript directories for given command.
+
+    Raises:
+        NotFoundError if the command could not be found.
+
+    Args:
+        cmd: The command to look for.
+
+    Returns:
+        A path to the userscript.
+    """
+    cmd_path = os.path.join(standarddir.data(), "userscripts", cmd)
+    if os.path.exists(cmd_path):
+        return cmd_path
```

Comment:
Why is this needed? Don't the lines below with the first entry in `directories` do the same?